revscores/languages/portuguese.py

```diff
@@ -1064,21 +1064,20 @@
     "zueira"
 ])
 
-class Portuguese(Language):
+def is_badword(word):
+    return STEMMER.stem(word).lower() in BADWORDS
+                
+def is_misspelled(word):
+    if word in ("a", "A"): return False
```

Comment:
This is hacky. Is "a" the only (single letter) word which causes problem?
("e" and "o" are also words in Portuguese)